src/components/NotesSection/NotesDisplay/NotesEditModal/NotesEditModal.jsx

```diff
@@ -0,0 +1,116 @@
+import TagsDisplay from "../../TagsDisplay/TagsDisplay";
+import { AiOutlinePushpin, AiFillPushpin } from "react-icons/ai";
+import NotesMenu from "../../NotesMenu/NotesMenu";
+import "./notesEditModal.css";
+import { useNotes } from "../../../../contexts";
+
+const NotesEditModal = ({
+  modalVisible,
+  updateModalVisibility,
+  tagId,
+  noteId,
+  title,
+  content,
+  bgColor,
+  tags,
+  isPinned,
+  deleteTag,
+  updateBgColor,
+  updateTagsList,
+}) => {
+  const { dispatch } = useNotes();
+
+  const updateDisplayNote = (id, property, value) => {
+    dispatch({
+      type: "UPDATE_NOTE_PROPERTY",
+      payload: { noteId: id, property, updatedValue: value },
+    });
+  };
+  
+  const updateInput = (event, type) => {
+    if (event.target.scrollHeight < 400) {
+      event.target.style.height = "inherit";
+      event.target.style.height = `${event.target.scrollHeight}px`;

```

Comment:
Instead of writing event.target.style. you can assign it to a variable and it would have made life easy :)